java/Makefile

```diff
@@ -437,6 +437,9 @@ plume.jar: ${JAVA_AND_CPP_FILES} lib
 	(cd jar-contents; find * -type f | ${PLUME_DIR}/bin/sort-directory-order > jar-contents.txt)
 # Temporarily commented out so that plume-lib can be built with Java 8;
 # but I need to find a good way to double-check most builds.
+# TODO: The above comment isn't clear - what has been commented out?
```

Comment:
You can remove those two lines, and your added lines.
Those two lines were not removed by commit 7f515922b4ed0eaa3c9c4cc69896c94ab720c517 as they should have been.  Thanks for noticing this.